plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPlugin.kt

```diff
@@ -36,83 +36,119 @@ class SentryPlugin : Plugin<Project> {
-                    val preBundleTask = withLogging("preBundleTask") {
-                        getPreBundleTask(project, variant.name)
+                val assembleTask = withLogging("assembleTask") {
+                    getAssembleTask(variant)
+                }
+
+                val sentryProperties = getPropertiesFilePath(project, variant)
+
+                val isMinifyEnabled = variant.buildType.isMinifyEnabled
+
+                var dexTask: Task? = null
+                var preBundleTask: Task? = null
+                var transformerTask: Task? = null
+                var packageTask: Task? = null
+                var mappingFile: File? = null
+                val sep = File.separator
+
+                if (isMinifyEnabled) {
+                    dexTask = withLogging("dexTask") {
+                        getDexTask(project, variant.name)
                     }
-                    val bundleTask = withLogging("bundleTask") {
-                        getBundleTask(project, variant.name)
+                    preBundleTask = withLogging("preBundleTask") {
+                        getPreBundleTask(project, variant.name)
                     }
-                    val transformerTask = withLogging("transformerTask") {
+                    transformerTask = withLogging("transformerTask") {
                         getTransformerTask(project, variant.name)
                     }
-                    val packageTask = withLogging("packageTask") {
+                    packageTask = withLogging("packageTask") {
                         getPackageTask(project, variant.name)
                     }
-                    val assembleTask = withLogging("assembleTask") { getAssembleTask(variant) }
-
-                    val sentryProperties = getPropertiesFilePath(project, variant)
-
-                    val extraProperties = project.extensions.getByName("ext")
-                        as ExtraPropertiesExtension
-
-                    val sentryOrgParameter = runCatching {
-                        extraProperties.get(SENTRY_ORG_PARAMETER).toString()
-                    }.getOrNull()
-                    val sentryProjectParameter = runCatching {
-                        extraProperties.get(SENTRY_PROJECT_PARAMETER).toString()
-                    }.getOrNull()
+                    mappingFile = getMappingFile(project, variant)
+                } else {
+                    project.logger.info(
+                        "[sentry] isMinifyEnabled is false for variant ${variant.name}."
+                    )
+                }
 
-                    val mappingFile = getMappingFile(project, variant)
-                    val cliExecutable = getSentryCliPath(project)
-                    val sep = File.separator
+                variant.outputs.all { variantOutput ->
+                    val taskSuffix = variant.name.capitalize(Locale.ROOT) +
+                        variantOutput.name.capitalize(Locale.ROOT)
```

Comment:
could use string templates here "${var}"

Reply:
Yup fixed

Reply:
Actually not, using string templates will lead the formatter to produce this:

```
val taskSuffix =
    "${variant.name.capitalize(Locale.ROOT)}${variantOutput.name.capitalize(
        Locale.ROOT
    )}"
```

That is not great. I'd rather stick to the `+` here